sktime/forecasting/base/_fh.py

```diff
@@ -466,7 +471,12 @@ def is_all_in_sample(self, cutoff=None):
     def _is_out_of_sample(self, cutoff=None):
         """Get index location of out-of-sample values."""
         # return ~self._in_sample_idx(cutoff)
-        return self.to_relative(cutoff).to_pandas() > 0
```

Comment:
redundant - should be simply `return not _is_in_sample(self, cutoff=cutoff)`, no?

Reply:
correct! but it is not `not`, but rather `np.logical_not`. too bad that there are no type suggestions, otherwise I would have spent much less time to figure out why a simple `not` does not work...

Reply:
ah... sorry to hear 😄